### PR TITLE
[SEM-043]: LABEL “ARTICULOS DESTACADOS”

### PR DESCRIPTION
**Pull request by 11royer**
Para que como usuario ver un label indicando artículos destacados.

### CODE REVIEW

**Comment by DanX3N0** [comment]
repair the pipe is failing

**Comment by DanX3N0** [comment]
FIGMA:
image

PR:
image
image

check the typography prop, it's no according to figma.

**Comment by DanX3N0** [comment]
PR:
image
image

FIGMA:
image

it's according to figma

File: src/ui/components/atoms/label-articulos-destacados/LabelArtDestacados.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import type { StoryObj, Meta } from '@storybook/react';
+import LabelArtDestacados from './LabelArtDestacados';
+import '../../../../globals/_variables.scss';
+
+const meta: Meta<typeof LabelArtDestacados> = {
+  title: 'ui/components/atoms/label-articulos-destacados',
```
Comment (Ditmar) [inline src/ui/components/atoms/label-articulos-destacados/LabelArtDestacados.stories.tsx:6]: please rename the name use English

**Review by Ditmar — CHANGES_REQUESTED** [review]
Use english

File: src/ui/components/atoms/label-articulos-destacados/LabelArtDestacados.stories.tsx (hunk 1 above)
Comment (11royer) [reply src/ui/components/atoms/label-articulos-destacados/LabelArtDestacados.stories.tsx]: buenas tardes ingeniero, ya corregi lo que me dijo

**Review by Ditmar — APPROVED** [review]

**Pull request merged**